shared/src/authorization/authorizationClientService.test.js

```diff
@@ -11,42 +11,81 @@ chai.use(require('chai-string'));
 
 describe('Authorization client service', () => {
   it('should authorize a petitions clerk for getCasesByStatus', () => {
-    expect(isAuthorized('petitionsclerk', GET_CASES_BY_STATUS)).to.be.true;
+    expect(
+      isAuthorized(
+        { userId: 'petitionsclerk', role: 'petitionsclerk' },
+        GET_CASES_BY_STATUS,
+      ),
+    ).to.be.true;
   });
 
   it('should authorize a user who is owner always', () => {
-    expect(isAuthorized('someUser', 'unknown action', 'someUser')).to.be.true;
+    expect(
+      isAuthorized(
+        { userId: 'someUser', role: 'petitioner' },
+        'unknown action',
+        'someUser',
+      ),
+    ).to.be.true;
   });
 
   it('should authorize a petitionsclerk for getCase', () => {
-    expect(isAuthorized('petitionsclerk', GET_CASE)).to.be.true;
+    expect(
+      isAuthorized(
+        { userId: 'petitionsclerk', role: 'petitionsclerk' },
+        GET_CASE,
+      ),
+    ).to.be.true;
   });
 
   it('should authorize a intakeclerk for getCase', () => {
-    expect(isAuthorized('intakeclerk', GET_CASE)).to.be.true;
+    expect(
+      isAuthorized({ userId: 'intakeclerk', role: 'intakeclerk' }, GET_CASE),
+    ).to.be.true;
   });
 
   it('should return false when a user doesnt have a petitionsclerk role', () => {
-    expect(isAuthorized('notaclerk', GET_CASES_BY_STATUS)).to.be.false;
+    expect(
+      isAuthorized(
+        { userId: 'intakeclerk', role: 'petitioner' },
```

Comment:
I'll just change the userId to something else, such as "someUser" to avoid confusion.  But yes, if they are an intakeclerk, they should never be a petitioner.